releng/releng-target/lsp4j.target.target

```diff
@@ -3,9 +3,9 @@
 <target name="org.eclipse.xtext.helios.target" sequenceNumber="0">
 	<locations>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tools/orbit/downloads/2021-09"/>
-			<unit id="com.google.gson" version="2.8.7.v20210624-1215"/>
-			<unit id="com.google.gson.source" version="2.8.7.v20210624-1215"/>
+			<repository location="https://download.eclipse.org/tools/orbit/downloads/2022-03"/>
```

Comment:
Use `https://download.eclipse.org/tools/orbit/downloads/latest-I/` for now and we can change it later (add it to the checklist on https://github.com/eclipse/lsp4j/issues/541 so it isn't forgotten)

Reply:
i would simply wait the 2 weeks for m2